apps/admin-panel/src/components/forms/field-group.tsx

```
@@ -0,0 +1,177 @@
+          />
+        )}
+      </div>
+      {error && (
+        <motion.p
+          id={`${inputId}-error`}
+          initial={{ opacity: 0, y: -10 }}
+          animate={{ opacity: 1, y: 0 }}
+          className="text-destructive flex items-center gap-1 text-sm"
+          role="alert"
+        >
+          <svg className="h-4 w-4" fill="currentColor" viewBox="0 0 20 20">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)